go/vt/sqlparser/ast_funcs.go

```diff
@@ -2536,3 +2540,14 @@ func IsLiteral(expr Expr) bool {
 func (ct *ColumnType) Invisible() bool {
 	return ct.Options.Invisible != nil && *ct.Options.Invisible
 }
+
+func (node *Delete) isSingleAliasExpr() bool {
+	if len(node.Targets) > 1 {
+		return false
+	}
+	if len(node.TableExprs) != 1 {
+		return false
+	}
```

Comment:
nit: you can move these two conditions into a single if

Reply:
will make the change in the next PR.